### ISSUE TITLE
Poor performance because of bad initialization

### ISSUE THREAD

**Issue by JerryPeng21cuhk** [RuiShu/vae-clustering#10]
This is for those are trying this vae-clustering repo which depends the tensorbayes library. I got very poor performance because of the bad initialization scheme in file: tensorbayes/layers/simple.py line 31.

Pls replace **variance_scaling_initializer** with **xavier_initializier**. Also, don't forget to import it: **from tensorflow.contrib.layers import xavier_initializer**

After this, the performance is boosted.
Hope this help later followers~

**Comment by JerryPeng21cuhk** [comment]
I implemented a pytorch version and found the clustering accuracy(Or purity ?) can be boosted to 95% after 738 epoches...
